packages/shorebird_cli/lib/src/commands/release/release_command.dart

```diff
@@ -22,6 +22,27 @@ class ReleaseCommand extends ShorebirdCommand {
   @override
   String get name => 'release';
 
+  static void printConflictingFlutterRevisionError({
+    required String existingFlutterRevision,
+    required String currentFlutterRevision,
+    required String releaseVersion,
+  }) {
+    logger.err(
+      '''
+${styleBold.wrap(lightRed.wrap('A release with version $releaseVersion already exists but was built using a different Flutter revision.'))}
+
+  Existing release built with: ${lightCyan.wrap(existingFlutterRevision)}
+  Current release built with: ${lightCyan.wrap(currentFlutterRevision)}
+
+${styleBold.wrap(lightRed.wrap('All releases for the given version must be built using the same Flutter version.'))}
```

Comment:
I think you mean "All platforms for a given release must use the same Flutter version."?